nbdime/__main__.py

```diff
@@ -7,34 +7,46 @@
 from __future__ import print_function
 
 import sys
+from subprocess import call
+
+try:
+    from shutil import which
+except ImportError:
+    from backports.shutil_which import which
+
+COMMANDS = ["show", "diff", "merge", "patch", "diff-web", "merge-web", "mergetool"]
 
 
 def main_dispatch(args=None):
     if args is None:
         args = sys.argv[1:]
     if len(args) < 1:
-        sys.exit("Please specify an nbdime command to call, such as 'nbdiff' or 'nbmerge'")
+        sys.exit("Command missing, expecting one of: \n%s" % ", ".join(COMMANDS))
 
     cmd = args[0]
     args = args[1:]
 
-    if cmd == "nbshow":
+    if cmd == "show":
         from nbdime.nbshowapp import main
-    elif cmd == "nbdiff":
+    elif cmd == "diff":
         from nbdime.nbdiffapp import main
-    elif cmd == "nbmerge":
+    elif cmd == "merge":
         from nbdime.nbmergeapp import main
-    elif cmd == "nbpatch":
+    elif cmd == "patch":
```

Comment:
Maybe remove patch here. We're not currently planning on implementing fuzzy patching, so it's not very useful and doesn't behave as users would expect from outside diff/patch experience.